### DISCUSSION TITLE
Flux styles for file inputs

### DISCUSSION THREAD

**Discussion by jeppeolesen** [livewire/flux#223] [Feature Requests] [answered]
Currently there is no styling applied for `<flux:input type="file" />`. 

<img width="561" alt="Screenshot 2024-10-01 at 19 41 38" src="https://github.com/user-attachments/assets/4ee414f4-48cc-423b-a5d7-82de9232000b">

There are workarounds using a hidden input and alpine/js, but it would be much cleaner if Flux supported file inputs.

**Comment by t-prod** [comment]
@jeppeolesen totaly agree with you

**Comment by dudelisius** [comment]
Would love to see this is as well, just styling would be good. Feels like the only missing form element

**Comment by calebporzio** [comment, marked as answer]
We just cranked out a few design ideas for this. Will be implementing at least a better looking file upload button soon.

**Reply by t-prod** [reply]
Hi Caleb,
Glad to see it soon ;)
